src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -95,6 +102,13 @@ def run(
         :return: The result of this batch run
         :rtype: ~promptflow.executor._result.BatchResult
         """
+        # Add property on operation context to indicate batch run source, which is used to differentiate the input
+        # source of a batch run. The batch run source can be either "Data" or "Run".
+        # If the input source is "Data", it means the input data is provided by the user.
+        # If the input source is "Run", it means the input data is provided by a previous run.
```

Comment:
Then how about naming them as something like `UserProvided` and `PreviousRun`?

Reply:
This is to align with name in extension `Run against run` and `Run against data`.